apps/public-web/src/EShopOnAbp.PublicWeb/ServiceProviders/PaymentTypeProvider.cs

```diff
@@ -9,15 +9,15 @@ public List<PaymentType> GetPaymentTypes()
     {
         return new List<PaymentType>
         {
-            new() {Id = 0, Name = "Demo", IconCss = "fa-credit-card demo", IsDefault = true},
```

Comment:
I think we should keep id and use name for type instead of keeping same string value for both properties.